spark/src/test/scala/geotrellis/spark/summary/polygonal/PolygonalUtils.scala

```diff
@@ -16,16 +16,20 @@
 
 package geotrellis.spark.summary.polygonal
 
-import geotrellis.vector._
-import geotrellis.vector.summary.polygonal._
+import cats.Monoid
 
-trait PolygonalSummaryFeatureCollectionMethods[G <: Geometry, D] {
-  val featureCollection: Seq[Feature[G, D]]
-
-  def polygonalSummary[T](polygon: Polygon, zeroValue: T)(handler: PolygonalSummaryHandler[G, D, T]): T =
-    featureCollection.aggregate(zeroValue)(handler.mergeOp(polygon, zeroValue), handler.combineOp)
-
-  def polygonalSummary[T](multiPolygon: MultiPolygon, zeroValue: T)(handler: PolygonalSummaryHandler[G, D, T]): T =
-    featureCollection.aggregate(zeroValue)(handler.mergeOp(multiPolygon, zeroValue), handler.combineOp)
+import scala.reflect.ClassTag
 
+trait PolygonalUtils {
+  /**
+    * Test helper to construct a Monoid[Array[T]] given a Monoid[T]
+    */
+  def arrayMonoidT[T : ClassTag](m: Monoid[T]): Monoid[Array[T]] = new Monoid[Array[T]] {
```

Comment:
Building from https://github.com/CloudNiner/geotrellis/pull/2#discussion_r294502919, this helper method is potentially but not immediately useful for those wanting to build their own operations on `Raster[Tile|MultibandTile]`. It should probably move with any monoid implementations rather than remain here as an awkward test helper.

Reply:
We def should not be making these but relying on `cats.implicits._` to implicitly build `Monoid[Array[T]]` instances

Reply:
After a bit of research, I'm not sure this is possible, as for all these `Monoid[Array[T]]` instances we want to do an elementwise combination rather than by just combining lists together which is the default implementation. So I left custom implementations of these in the companion object for each value type. Anyone who wants to implement a visitor for a MultibandTile will need to think about how they're combining their bands anyways.